bin/configs/kotlin-spring-boot-reactive-without-flow.yaml

```diff
@@ -0,0 +1,13 @@
+generatorName: kotlin-spring
+outputDir: samples/server/petstore/kotlin-springboot-reactive-without-flow
+library: spring-boot
+inputSpec: modules/openapi-generator/src/test/resources/3_0/petstore.yaml
+templateDir: modules/openapi-generator/src/main/resources/kotlin-spring
+additionalProperties:
+  documentationProvider: springdoc
+  annotationLibrary: swagger2
+  useSwaggerUI: "true"
+  serviceImplementation: "true"
+  reactive: "true"
+  beanValidations: "true"
+  useFlowForArrayReturnType: "false"
```

Comment:
i think this already covered by the test in samples/server/petstore/kotlin-springboot-reactive so i'll remove it after merging this pr